src/barChart.ts

```diff
@@ -444,6 +460,12 @@ export class BarChart implements IVisual {
             .remove();
 
         this.handleClick(barSelectionMerged);
+
+        if (/\d/.test(settings.warningIconExample.text)) {
```

Comment:
sorry - what is this syntax '/\d/' ?

Reply:
regex syntax - irrelevant after resolving comments